bosh_troubleshooting.prolific

```diff
@@ -6,10 +6,11 @@ To SSH into a BOSH job, you'll use the `bosh` cli.
 ### How?
 bosh-bootloader sets up a jumpbox that allows `bosh` commands to be run from your workstation
 * `eval "$(bbl print-env)"` to target the director
-* `bosh ssh` should just work with no further information needed from bbl.
+* `bosh ssh` should just work with no further information needed from bbl. You still need the guid and name of the VM
+you SSH into. 
```

Comment:
It might be clearer to rewrite the sentence a bit more, it seems like "...should just work with no further information needed..." is misleading. e.g. "Now, with only the VM's GUID `bosh ssh` will connect to your VM."